corehq/apps/app_manager/templates/app_manager/apps_base.html

```diff
@@ -18,6 +18,23 @@
     {% endcompress %}
 {% endblock %}
 
+{% block head %}{{ block.super }}
```

Comment:
I think you could make this a template and include it on each page that is pertinent